lib/known_bugs.pm

```diff
@@ -28,12 +28,21 @@ our @EXPORT = qw(
   upload_journal
 );
 
+=head1 SYNOPSIS
+
+Allow detection of known errors on the serial console
+=cut
+
+=head2 create_list_of_serial_failures
+
+Creates the list of known bug patterns on the serial logs
+This will be used in main.pm to initialize the backend with that list
+To add a known bug simply copy and adapt the following line:
+C<push @$serial_failures, {type => soft/hard, message => 'Errormsg', pattern => quotemeta 'ErrorPattern' }>
+=cut
```

Comment:
For retro compatibility, we need to add a space before cut.

```suggestion

=cut
```

https://perldoc.perl.org/perlpod.html#Hints-for-Writing-Pod